docs/getting-started/contributing/contributing-with-git.md

```diff
@@ -0,0 +1,196 @@
+
+    | System  | Installation                                                                                                                                                                                               |
+    |---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Windows | Download and install git for Windows from the [git-scm website (external)](https://git-scm.com/download/win).  If you have winget you may install git via `winget install --id Git.Git -e --source winget` |
+    | Mac     | Mac installs git automatically when you run it from the terminal.  If you have Homebrew, you may install git via `brew install git`.                                                                       |
+    | Linux   | Install git using your package manager. For example, `sudo apt install git` on Ubuntu.                                                                                                                     |
+
+
+```sh
+# Once git is installed, set your name and email:
+git config --global user.name "Your Name"
+git config --global user.email "emailaddress@example.com"
+```
+
+### CONTRIBUTING.md
+
+Some projects have a `CONTRIBUTING.md` file in their repo.  It can help explain that project's contribution guidelines, and how you can make your contribution most effective.
+
+## Issues
+
+![Issue board on GitHub](../../assets/software-management/git-issue-board.png){ align=right width=450 }
+
+Your contribution should be related to an issue.  You can browse a list of issues on the repository page for each project. 
+
+A git issue is not necessarily something wrong, it may also be a feature to be added.
+
+!!! tip
+
+    If you are working on an issue you have found yourself, and the issue is not listed yet, please [report the issue](../../how-to-guides/software-management/guidelines-for-reporting-issues.md) and assign it to yourself.
+
+-----
+
+## Fork
+
+![Forking a repository on GitHub](../../assets/software-management/git-fork.png){ align=right width=450 }
+
+A ***fork*** is a copy of a repository.  The repo that has been forked from is usually called ***"upstream"***, and your fork/working copy is called ***"origin"*** by default.
+
+Your fork is your own working copy, where you can test your code or add new features.  While you are working on your contribution, you don't have to worry about unfinished code affecting the upstream repo; you are on your own fork.  Later, when your code is done, there is a review process before your code can be merged into the upstream repo.
+
+#### To make a fork:
+
+1. Click the fork button on the repository page
+1. Enter a name for your fork.
+    - If you want to contribute back to the original repo, you can just leave the name the same (your fork is differentiated by your username).
+
+-----
+
+## Clone
+
```

Comment:
Would it make sense to simply offload this section to [the GitHub docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository). Then the screen capture can go and we need only maintain the link (and not this content). If that is done, then perhaps it makes sense to include a sentence or two describing why someone is performing a clone as part of getting started contributing.